File: app/src/main/java/com/zawzaw/savethelibrary/service/FirebaseIdService.java
package com.zawzaw.savethelibrary.service;

import android.util.Log;
import com.google.firebase.iid.FirebaseInstanceId;
import com.google.firebase.iid.FirebaseInstanceIdService;

/**
 * Created by zawzaw on 30/01/18.
 */

public class FirebaseIdService extends FirebaseInstanceIdService {

    public FirebaseIdService() {

    }

    @Override
    public void onTokenRefresh() {
        String refreshToken = FirebaseInstanceId.getInstance().getToken();
        Log.i("TOKEN", refreshToken);
    }

}

File: app/src/main/java/com/zawzaw/savethelibrary/service/FirebaseIdService.kt
package com.zawzaw.savethelibrary.service

import android.util.Log
import com.google.firebase.iid.FirebaseInstanceId
import com.google.firebase.iid.FirebaseInstanceIdService

/**
 * Created by zawzaw on 30/01/18.
 */

class FirebaseIdService : FirebaseInstanceIdService() {

    override fun onTokenRefresh() {
        val refreshToken = FirebaseInstanceId.getInstance().token
        Log.i("TOKEN", refreshToken)
    }

}
